### PR TITLE
HSSS CUSP3D to TitanSMA SM Recorder Swap

### PR DESCRIPTION
**Pull request by Thomas-Benson**
- Removing Cusp3D (42025) from HSSS LC 20
- Replacing with TitanSMA (002024) at same location
- Using latest TitanSMA firmware version 4.5.22
- Updated streams to continuous from 2020-02-09
- Streams affected HSSS LC 20 HN-12Z

### CODE REVIEW

File: install/recorders.csv
Comment (mnaguit) [inline install/recorders.csv:1]: not to be fussy, for consistency: opening is `2023-11-13T21:29:00Z` (instead of `2023-11-13T21:29:10Z`)

**Review by mnaguit — CHANGES_REQUESTED** [review]
very minor tweak on the opening time

File: install/recorders.csv
Comment (Thomas-Benson) [reply install/recorders.csv:1]: Fixed now, sorry for the typo.

Comment (mnaguit) [reply install/recorders.csv:1]: No apologies, you are doing good! 👍

**Review by mnaguit — APPROVED** [review]

**Review by CBurton90 — APPROVED** [review]

**Pull request merged**